MHacks/AnnouncementsViewController.swift

```diff
@@ -97,16 +97,21 @@ class AnnouncementsViewController: UITableViewController {
     }
 	
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        
         let cell = tableView.dequeueReusableCell(withIdentifier: "item", for: indexPath) as! AnnouncementCell
-        
+
         let announcement = APIManager.shared.announcements[(indexPath as NSIndexPath).row]
-        
+		
         cell.titleLabel.text = announcement.title
         cell.dateLabel.text = announcement.localizedDate
         cell.messageLabel.text = announcement.message
 		cell.colorView.layer.borderColor = announcement.category.color.cgColor
 		cell.colorView.layer.borderWidth = cell.colorView.frame.width
+		
+		if !announcement.isSponsored {
+			cell.contentStackView.removeArrangedSubview(cell.sponsoredLabel)
+			cell.sponsoredLabel.removeFromSuperview()
```

Comment:
Just like the below comment, thats what I did first but it didn't work :/ I just tried it again and there is noticeably more space being rendered with
`cell.sponsoredLabel.isHidden = !announcement.isSponsored` rather than lines 110 - 113...

I don't know if that means that `UIStackView` only _updates_ the layout when the view is hidden rather than if the view is hidden before the contents of the stack view are even rendered? but that doesn't really make sense why so maybe I'm just doing something dumb